core/src/test/java/org/apache/calcite/plan/RelWriterTest.java

```diff
@@ -658,14 +658,25 @@
     final RelJsonWriter jsonWriter = new RelJsonWriter();
     rel.explain(jsonWriter);
     final String relJson = jsonWriter.asString();
-    String s = deserializeAndDumpToTextFormat(getSchema(rel), relJson);
-    final String expected = ""
+    String s = deserializeAndDumpToTextFormat(getSchema(rel), relJson, format);
+    assertThat(s, isLinux(expected));
+  }
+
+  @Test void testAggregateWithAlias() {
+    testAggregateWithAlias0(SqlExplainFormat.TEXT, ""
         + "LogicalProject(max_sal=[$1])\n"
         + "  LogicalAggregate(group=[{0}], max_sal=[MAX($1)])\n"
         + "    LogicalProject(JOB=[$2], SAL=[$5])\n"
-        + "      LogicalTableScan(table=[[scott, EMP]])\n";
+        + "      LogicalTableScan(table=[[scott, EMP]])\n");
 
-    assertThat(s, isLinux(expected));
+    testAggregateWithAlias0(SqlExplainFormat.DOT, "digraph {\n"
+        + "\"LogicalProject(max_s\\nal=[$1])\" -> \"LogicalAggregate(gro\\nup=[{0}], "
+        + "max_sal=[M\\nAX($1)])\" [label=\"0\"]\n"
+        + "\"LogicalAggregate(gro\\nup=[{0}], max_sal=[M\\nAX($1)])\" -> \"LogicalProject"
```

Comment:
Text such as `LogicalAggregate(gro\\nup=[{0}]` and `LogicalTableScan(tab\\nle=` is a little weird. Maybe we can change it to LogicalAggregate\n(group=[{0}]`. What do you think?

Reply:
Sounds reasonable. It seems @danny0405 has a similar suggestion.
I have revised the PR to fix this.